### PR TITLE
Setup animation controls

### PR DESCRIPTION
**Pull request by tealefristoe**
- Add and style control toggle widget.
- Display proper controls for all simulations and animations.
- Setup infrastructure to make correct videos display based on control selections.
- Various styling improvements.

### CODE REVIEW

File: src/components/simulation-app.tsx [hunk 1]
```diff
@@ -16,78 +20,58 @@ export const SimulationApp = ({ ac }: SimulationAppProps) => {
...
+  const [control2, setControl2] = useState(false);
+  const [control3, setControl3] = useState(false);
+  useEffect(() => {
+    console.log(`${ac.o("SIMCONTROL1LABEL")}: ${ac.o(control1 ? "SIMCONTROL1OPTION2" : "SIMCONTROL1OPTION1")}`);
+  }, [ac, control1]);
+  useEffect(() => {
+    console.log(`${ac.o("SIMCONTROL2LABEL")}: ${ac.o(control2 ? "SIMCONTROL2OPTION2" : "SIMCONTROL2OPTION1")}`);
+  }, [ac, control2]);
+  useEffect(() => {
+    console.log(`${ac.o("SIMCONTROL3LABEL")}: ${ac.o(control3 ? "SIMCONTROL3OPTION2" : "SIMCONTROL3OPTION1")}`);
```
Comment (bacalj) [inline src/components/simulation-app.tsx:34]: Are these here intentionally for the moment?

File: src/components/animation-app.tsx [hunk 2]
```diff
@@ -45,18 +56,26 @@ export const AnimationApp = ({ ac }: AnimationAppProps) => {
             ac={ac}
             playing={playingTissue}
             setPlaying={setPlayingTissue}
-            title={ac.o("LEFTANIMATIONTITLE")}
```
Comment (bacalj) [inline src/components/animation-app.tsx:48]: Do you want `disabled={playingCell}` on the tissue video? Right now behaviors don't match.

**Review by bacalj — APPROVED** [review]
Looks good! 

I noticed one layout issue that is not connected to these changes, so I'm putting it in this comment.  

I notice that if the paragraph text in the right box under stress input grows longer than the predicted height, everything breaks out of the box.   I tried the change below and it allowed the app container to grow if the app grows:

`app.scss`, line 20, change height to min-height:

```scss

.app-container {
    min-height: 598px;
}
```
If the row grows too tall, the silhouette will have weird space below, so you could give it:
```scss
.silhouette {
   justify-self: center
}
```
Anyway one other small thing I noticed - with the disable video thing - which is in the inline comments.

File: src/components/simulation-app.tsx (hunk 1 above)
Comment (tealefristoe) [reply src/components/simulation-app.tsx:34]: Yeah these are just temporary to show that the toggles are toggling. I've got a PT story to remove them when we have the other videos to make the toggles work as intended.

File: src/components/animation-app.tsx (hunk 2 above)
Comment (tealefristoe) [reply src/components/animation-app.tsx:48]: It's weird, but it's intended that they work asymmetrically. We'll see what the PIs think once they realize how things actually work based on their design.

**Pull request merged**